src/Models/SiteTreeLink.php

```diff
@@ -134,4 +134,19 @@ public function getTitle(): ?string
         // Use page title as a default value in case CMS user didn't provide the title
         return $page->Title;
     }
+
+    public function getLinkTypeDefaultTitle(): string
+    {
+        $pageExist = $this->Page()->exists() && $this->getURL();
```

Comment:
```suggestion
        $pageExist = $this->Page()->exists();
```
This should produce the same result, shouldn't it? If not, please add a code comment explaining why we _also_ need to check `getURL()` here.

Reply:
Updated